packages/@webex/plugin-meetings/test/unit/spec/meeting/utils.js

```diff
@@ -740,5 +740,72 @@ describe('plugin-meetings', () => {
         assert.deepEqual(MeetingUtil.isBreakoutPreassignmentsEnabled([]), true);
       });
     });
+
+
+    describe('parseInterpretationInfo', () => {
+      let meetingInfo = {};
+
+      it('should update simultaneous interpretation settings for the meeting', () => {
+        meeting.simultaneousInterpretation = {
+          updateMeetingSIEnabled: sinon.stub(),
+          updateHostSIEnabled: sinon.stub(),
+          updateInterpretation: sinon.stub(),
+          siLanguages: [],
+        };
+
+        meetingInfo.turnOnSimultaneousInterpretation = true;
+        meetingInfo.meetingSiteSetting = {
+          enableHostInterpreterControlSI: true,
+        };
+        meetingInfo.simultaneousInterpretation = {
+          currentSIInterpreter: true,
+          siLanguages: [
+            { languageCode: 'en', languageGroupId: 1 },
+            { languageCode: 'es', languageGroupId: 2 },
+          ],
+        };
+
+        MeetingUtil.parseInterpretationInfo(meeting, meetingInfo);
+        assert.calledWith(meeting.simultaneousInterpretation.updateMeetingSIEnabled, true, true);
+        assert.calledWith(meeting.simultaneousInterpretation.updateHostSIEnabled, true);
+        assert.calledWith(meeting.simultaneousInterpretation.updateInterpretation, [
+          { languageName: 'en', languageCode: 1 },
+          { languageName: 'es', languageCode: 2 },
+        ]);
+
+        meetingInfo.meetingSiteSetting.enableHostInterpreterControlSI = false;
+        meetingInfo.simultaneousInterpretation.currentSIInterpreter = false;
+        MeetingUtil.parseInterpretationInfo(meeting, meetingInfo);
+        assert.calledWith(meeting.simultaneousInterpretation.updateMeetingSIEnabled, true, false);
+        assert.calledWith(meeting.simultaneousInterpretation.updateHostSIEnabled, false);
+        assert.calledWith(meeting.simultaneousInterpretation.updateInterpretation, [
+          { languageName: 'en', languageCode: 1 },
+          { languageName: 'es', languageCode: 2 },
+        ]);
+
+
+        meetingInfo.turnOnSimultaneousInterpretation = false;
+        MeetingUtil.parseInterpretationInfo(meeting, meetingInfo);
+        assert.calledWith(meeting.simultaneousInterpretation.updateMeetingSIEnabled, false, false);
+        assert.calledWith(meeting.simultaneousInterpretation.updateHostSIEnabled, false);
```

Comment:
issue: This asset will work even if you don't call parseInterpretationInfo again, because you haven't reset the stubs in between checks. 

In this case it would be much better to write three separate tests, either using it.each or just split completely. You could write a shared setup function to avoid setting up the stubs and the data each time.

Reply:
Modified as your suggestion.